HerPublicWebsite/Views/Questionnaire/SelectLocalAuthority.cshtml

```diff
@@ -0,0 +1,75 @@
+﻿@using GovUkDesignSystem
+@using GovUkDesignSystem.GovUkDesignSystemComponents
+@using HerPublicWebsite.Controllers
+
+@model HerPublicWebsite.Models.Questionnaire.SelectLocalAuthorityViewModel
+@{
+    ViewBag.Title = "Select your local authority";
+}
+
+@section BeforeMain {
+    @await Html.GovUkBackLink(new BackLinkViewModel
+           {
+               Text = "Back",
+               Href = Model.BackLink,
+           })
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        
+        <h1 class="govuk-heading-l">We couldn't find your local authority</h1>
+        <p class="govuk-body">Click your local authority in the list below.</p>
+        
+        <div class="her_alphabet_links">
+            @{
+                var orderedLetters = Model.LocalAuthoritiesByInitial.Keys.OrderBy(k => k).ToList();
+                var lettersInFirstRow = (orderedLetters.Count + 1) / 2;
```

Comment:
I don't know if it's something you've discussed, but I wonder if it would be easier to display all letters and disable the ones with no results? This is probably fine for beta anyway.

Reply:
Yeah, I can't quite decide which is more confusing - I could also imagine low-literacy users struggling if there's a seemingly random subset of letters. It probably doesn't matter because in practice they're almost all there - let's see what the channel says!